edge-modules/MetricsCollector/MetricsCollector.csproj

```diff
@@ -31,6 +31,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\edge-util\src\Microsoft.Azure.Devices.Edge.Util\Microsoft.Azure.Devices.Edge.Util.csproj" />
+    <ProjectReference Include="..\..\test\modules\ModuleLib\Microsoft.Azure.Devices.Edge.ModuleUtil.csproj" />

```

Comment:
FYI, once this project is officially released like tempSensor; we need to refactor this dependency.

Reply:
I talked with Venkat and we are never going to do this, because we want Azure Monitor to own it.